src/main/java/frc/robot/commands/TankDrive.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import frc.robot.Robot;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class TankDrive extends CommandBase {
+
+  public TankDrive() {
+    addRequirements(Robot.driveTrain);
+  }
+
+  private void addRequirements(frc.robot.Constants.DriveTrain driveTrain) {
+}
+
+// Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+      Robot.driveTrain.setBothPrecent(Robot.joysticks.getLeftY(), Robot.joysticks.getRightY());
```

Comment:
For better code structure avoid accessing Robot objects such as joysticks. Create a clear structure:
Robot --registers--> Commands --call--> Subsystems --runs--> SpeedControllers
One way to do this is by creating a supplier like so:
```suggestion
   // in the constructor:
   public TankDrive(DoubleSupplier leftSupplier, DoubleSupplier rightSupplier) {
       this.leftSupplier = leftSupplier;
       this.rightSupplier = rightSupplier;
       ...
      Robot.driveTrain.setBothPrecent(leftSupplier.getAsDouble(), rightSupplier.getAsDouble());
```

where you create a DoubleSupplier from any function that returns a double like so:
```java
// in Robot.java
DoubleSupplier leftSupplier = joysticks::getLeftY;
DoubleSupplier rightSupplier = joysticks::getRightY;
... new TankDrive(leftSupplier, rightSupplier);
```